src/sections/Community/Meshmates/index.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import { Container, Row, Col } from "../../../reusecore/Layout";
+import Button from "../../../reusecore/Button";
+import ProfileCard from "../../../components/Profile-card";
+import MeshMatesWrapper from "./meshmates.style";
+import EmeritusGrid from "../Emeritus-grid";
+import { useStaticQuery, graphql } from "gatsby";
+
+import meshmate from "./meshmate.svg";
+import c_icon from "./c-icon.svg";
+import hero from "./hero.png";
+
+const Meshmates=() => {
+    const data=useStaticQuery(
+        graphql`
+            query meshmates {
+                allMdx(
+                    sort: { fields: [frontmatter___name], order: ASC }
+                    filter: { fields: { collection: { eq: "members" } }, frontmatter: { meshmate: { eq: "yes" } } }
+                ) {
+                    nodes {
+                        id
+                        frontmatter {
+                            name
+                            github
+                            twitter
+                            status
+                            linkedin
+                            location
+                            badges
+                            bio
+                            meshmate
+                            image_path{
+                                childImageSharp{
+                                    fluid(maxWidth: 200){
+                                        ...GatsbyImageSharpFluid
+                                    }
+                                }
+                                extension
+                                publicURL
+                            }
+                        }
+                        fields{
+                            slug
+                        }
+                    }
+                }
+            }
+        `
+    );
+    return (
+        <MeshMatesWrapper>
+            <div className="hero">
+                <Container>
+                    <div className="hero-text">
+                        <h4> A warm and welcoming collection of open sourcers </h4>
+                        <h1> Layer5 Meshmates </h1>
+                        <p> Layer5 MeshMates are committed to helping community members be successful contributors. MeshMates aid in identifying areas of projects to engage within, working groups to join, and in helping community members grow in their open source and cloud native knowledge. By connecting one-on-one, MeshMates will share tips on how to have the best community experience possible. </p>
+                    </div>
+                </Container>
+            </div>
+            <Container>
+                <div className="meshmates-grid">
+                    <h1> Layer5 MeshMates </h1>
```

Comment:
This should be removed or something different should be used as `Layer5 MeshMates` was already used in the hero